FrontEnd/src/BookingPage/BookingDB.js

```diff
@@ -65,22 +69,34 @@ const appointmentChanges = (userID) => {
     snapshot.docChanges().forEach((change) => {
       const appointmentData = change.doc.data();
       const vendorID = appointmentData.vendorUID;
+      console.log(appointmentData);
 
       if (change.type === "added") {
+        console.log("added doc");
         const notificationMessage = `You just requested an appointment on ${appointmentData.appointmentDate} with ${appointmentData.vendorUsername} for ${appointmentData.appointmentTitle}`;
         addNotification(userID, notificationMessage, appointmentData);
 
         const vendorNotificationMessage = `A new service request has been made by ${appointmentData.customerUsername} for ${appointmentData.appointmentTitle}`;
         addNotification(vendorID, vendorNotificationMessage, appointmentData);
       } else if (
         change.type === "modified" &&
-        appointmentData.Status === "accepted"
+        appointmentData.Status === ACCEPTED_STATUS
       ) {
+        console.log("modified to accepted");
```

Comment:
😼

Reply:
I couldn't go through the codes for review cause my laptop was turning off by itself and I didn't want to lose changes. Taking them out in the next commit!